src/main/kotlin/io/sip3/salto/ce/sip/SipMessageHandler.kt

```diff
@@ -194,19 +194,8 @@ open class SipMessageHandler : AbstractVerticle() {
     }
 
     open fun routeSipMessage(prefix: String, packet: Packet, message: SIPMessage) {
-        val route = when (prefix) {
-            RoutesCE.sip + "_call" -> {
-                val index = message.callId().hashCode()
-                prefix + "_${abs(index % instances)}"
-            }
-            RoutesCE.sip + "_register" -> {
-                // RFC-3261 10.2: The To header field contains the address of record
-                // whose registration is to be created, queried, or modified.
-                val index = message.toUri().hashCode()
-                prefix + "_${abs(index % instances)}"
-            }
-            else -> return
-        }
+        val index = message.callId().hashCode()
```

Comment:
We need to exclude some cseq methods from aggregation: 
1. There is no much sense to aggregate CANCEL transaction because we define state `Canceled` based on 487 status code of INVITE transaction.
2. ACK transaction is useless as well, plus sometimes it doesn't require response at all what makes it's aggregation the waste of time.